README.md

```diff
@@ -1,7 +1,14 @@
-A bzl file that contains maven_jar and maven_aar rules for all artifacts in
-https://maven.google.com. Not guaranteed to be correct or up to date. Some of
-the artifacts depend on artifacts that are not present on
-https://maven.google.com. We ignore these and hope not to fail.
+# gmaven_rules
+
+This project generates `gmaven.bzl`, a file containing `maven_jar` and `maven_aar`
+rules for all artifacts in [https://maven.google.com](https://maven.google.com).
+
+# Support Policy
+
+This project is an **interim solution** during which Google Maven and AAR
+support is added to [bazel-deps](https://github.com/johnynek/bazel-deps).
+
+# Usage instructions
```

Comment:
While you're here, could you add a sentence describing what to use for the <FILL IN A COMMIT HERE>? Suggestion, something like:

"We suggest you use the most recent commit from this repository."

Reply:
I was thinking of creating releases after migrating to bazelbuild. We can keep track of versions that way, instead of relying on unreadable hashes.

Reply:
SGTM